spec/features/filter_by_categories_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+RSpec.feature "filtering items by category", type: :feature do
+  scenario "it has a categories filter" do
+    visit items_path
+
+    within ".filter" do
+      expect(page).to have_content("Choose your food quest:")
+    end
+  end
+
+  context "interacting with the categories filter" do
+    before(:each) { Category.create(name: "pasta")
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.